packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Inbound/XCA ITI-38 Inbound Interface/sourceConnector-transformer-step-4-Get query parameters.js

```diff
@@ -41,6 +41,9 @@ try {
 		var func = queryRequestOptionFn[slot.@name.toString().slice(1)];
 		if (typeof func === 'function') {
 			var result = func(slot);
+			if (element === "entryPatientId") {
+				element = "externalGatewayPatient";
+			}
 			if (result) request[element] = result;
```

Comment:
As a general good practice, if we can code using constants instead of variables, it makes the code more robust and easier to understand.

Not a huge deal here, but this construct led me to look on the rest of the code to make sure we're not updating something that's used afterwards.

So can we create a new constant with the property we're going to access and initialize it according to the logic on the conditional/if? A ternary would do.